fastly/block_fastly_service_v1_director.go

```diff
@@ -30,14 +30,21 @@ func (h *DirectorServiceAttributeHandler) Process(d *schema.ResourceData, latest
 		nd = new(schema.Set)
 	}
 
-	ods := od.(*schema.Set)
-	nds := nd.(*schema.Set)
+	oldSet := od.(*schema.Set)
+	newSet := nd.(*schema.Set)
 
-	removeDirector := ods.Difference(nds).List()
-	addDirector := nds.Difference(ods).List()
+	setDiff := NewSetDiff(func(resource interface{}) (interface{}, error) {
```

Comment:
@phamann Hmm. Might tap you up Monday (or whenever you have time, if any) for a pairing session to help catch some test errors.